components/navigation.css

```
@@ -0,0 +1,23 @@
+.navigation {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+  background-color: var(--color-white);
+  padding: 12px 29px;
+  border-radius: 10px;
+  box-shadow: 0 2px 11px var(--shadow-grey);
+  position: fixed;
+  bottom: 20px;
+  left: 20px;
+  right: 20px;
+}
+
+.navigation a {
```

Maybe give a class here as well?
```suggestion
.navigation__link {
```